CHARTER.md

```diff
@@ -8,17 +8,18 @@ This Technical Charter sets forth the responsibilities and procedures for techni
 
 #### 1. Mission and Scope of the Technical Initiative
 
-- a. The mission of the Technical Initiative is to provide open source developers with best practices recommendations, and with an easy way to learn and apply them.
+The Best Practices for OSS Developers working group is dedicated to 
+- helping identify and curating an accessible inventory of best practices; 
+- raising awareness; 
+- and education of secure code best practices for open source developers.
 
-- b. The scope of the Technical Initiative includes collaborative development under the Technical Initiative License (as defined herein) supporting the mission, including organizing collaboration activities, defining best practices, documentation, testing, integration, and the creation of other artifacts that support the mission.
+#### 2. Technical initiative roles
 
-#### 2. Technical Steering Committee
+- a. The primary points of contact are:
```

Comment:
Don't enshrine names in the charter, only roles. Making changes to the charter is A Thing. Instead, put the names in the README and/or add a link directing people to the current holders of the listed roles.